src/backend/data/post.js

```diff
@@ -17,15 +17,20 @@ const { indexPost } = require('../utils/indexer');
 function ensureDate(date, fallbackDate) {
   if (
     date &&
-    (Object.prototype.toString.call(date) === '[object String]' ||
-      (Object.prototype.toString.call(date) === '[object Date]' && !Number.isNaN(date)))
+    ((Object.prototype.toString.call(date) === '[object String]' &&
```

Comment:
None of this needs to be modified, it's only the test that needs to be modified.